demo/kitchen_sink/run.py

```diff
@@ -151,10 +151,9 @@ def fn(
     * 3,
     theme="default",
     title="Kitchen Sink",
-    cache_examples=False,
     description="Try out all the components!",
     article="Learn more about [Gradio](http://gradio.app)",
-    cache_examples=True
```

Comment:
How come we need to not cache examples?

Reply:
Do we want to cache examples? I'm fine either way, but it's just a demo, right?

Reply:
Was thinking it could help catch accidental breaking changes when it comes to caching examples since this demo will run on spaces for prs.

Reply:
Ah yeah makes sense. Will remove the cached examples argument so it caches on Spaces